buildlib/jucx/jucx-build.yml

```diff
@@ -43,7 +46,7 @@ jobs:
           MAVEN_VERSION=${TAG:1}
           make -C bindings/java/src/main/native/ package JUCX_VERSION=${MAVEN_VERSION}
         displayName: Build JUCX
-        condition: eq(variables['Build.Reason'], 'PullRequest')
+        condition: eq(variables['Build.Reason'], 'IndividualCI')
```

Comment:
Is it for debug?

Reply:
This is to only run when the condition is right. 
[Sample skip log](https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=74349&view=logs&j=63c6b53b-67c9-50a3-0a67-f7e911956bf3&t=cf9a9368-6788-584f-400e-578b0d707aeb&l=1)